docs/operator-manual/airgap/01-install.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_label: "Installation"
+title: ""
+---
+
+# Airgap installation
+
+This guide will show you how to install Kubewarden in air-gapped environments. In an air-gapped installation of Kubewarden, 
+you will need a private OCI registry that is located somewhere accessible by your kubernetes cluster. Kubewarden Policies 
+are WebAssembly modules, therefore they can be stored inside of an OCI compliant registry as OCI artifacts.
+You need to add Kubewarden's images and policies to this OCI registry. Let's see how to do that.
+
+## Save container images in your workstation
+
+1. Download `kubewarden-images.txt` from the Kubewarden [release page](https://github.com/kubewarden/helm-charts/releases/). Alternatively, you can find `imagelist.txt` and `policylist.txt` files shipped inside the helm charts, that contain the used container images and policy wasm modules respectively.
```

Comment:
> Alternatively, you can find `imagelist.txt` and `policylist.txt` files shipped inside the helm charts

nitpicking: you hint about this other solution, but then there's no explanation about how to leverage that

Reply:
It is an alternative to downloading `kubewarden-images.txt`. You can either downlaod it from the release page or extract it from the helm chart. It was suggested [here](https://github.com/kubewarden/docs/pull/150#discussion_r1029032857)